src/main/java/org/spongepowered/api/statistic/Statistic.java

```diff
@@ -25,103 +25,30 @@
 package org.spongepowered.api.statistic;
 
 import org.spongepowered.api.CatalogType;
-import org.spongepowered.api.Sponge;
+import org.spongepowered.api.scoreboard.critieria.Criterion;
 import org.spongepowered.api.text.translation.Translatable;
-import org.spongepowered.api.text.translation.Translation;
-import org.spongepowered.api.util.ResettableBuilder;
 import org.spongepowered.api.util.annotation.CatalogedBy;
 
+import java.text.NumberFormat;
 import java.util.Optional;
 
-import javax.annotation.Nullable;
-
 /**
  * Represents some statistic in Minecraft with a string ID.
  */
 @CatalogedBy(Statistics.class)
 public interface Statistic extends CatalogType, Translatable {
 
     /**
-     * Creates a new {@link StatisticBuilder} to build a {@link Statistic}.
-     *
-     * @return The new builder
-     */
-    static Builder builder() {
-        return Sponge.getRegistry().createBuilder(Builder.class);
-    }
-
-
-    /**
-     * Gets the {@link StatisticFormat} of this statistic. If this is not
-     * present that this statistic's format is deferred to its group's default
-     * format.
-     *
-     * @return The format of this statistic, if available
-     */
-    Optional<StatisticFormat> getStatisticFormat();
-
-    /**
-     * Gets the {@link StatisticGroup} this {@link Statistic} belongs to.
+     * Returns the objective {@link Criterion} for this statistic.
      *
-     * @return The statistic group this statistic belongs to
+     * @return Objective criterion
      */
-    StatisticGroup getGroup();
+    Optional<Criterion> getCriterion();
 
-    interface Builder extends StatisticBuilder<Statistic, Builder> {
+    NumberFormat getFormat();
```

Comment:
Then you can also drop this NumberFormat for the same reasons.
Especially since its not very usable for all instances (time + distance).